### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by ronal2do**
https://github.com/prettier/prettier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +23,8 @@
     "bootstrap": "4.0.0-alpha.6",
     "compression": "^1.6.2",
     "express": "^4.14.0",
+    "husky": "^0.14.3",
```
Comment (YutHelloWorld) [inline package.json]: Maybe we should install them in devDependencies by  `yarn add lint-staged husky --dev`

**Comment by YutHelloWorld** [comment]
And we used [stantard](https://github.com/standard/eslint-config-standard) and [standard-react](https://github.com/standard/eslint-config-standard-react) in [.eslintrc](https://github.com/YutHelloWorld/vortex-react/blob/master/.eslintrc). I thought it's unnecessary to use prettier ?

File: package.json (hunk 1 above)
Comment (ronal2do) [reply package.json]: I often use air-bnb, I can adjust everything

**Comment by ronal2do** [comment]
I often use air-bnb, I can adjust everything

**Comment by YutHelloWorld** [comment]
Ok, this PR will be closed. BTW , thx for your contribution !

**Pull request closed without merging**